user_guide.md

```diff
@@ -411,14 +447,22 @@ tmpfs                                                           15444244       0
 On [Minikube](https://github.com/kubernetes/minikube) the Virtualbox/VMware drivers for Minikube are recommended as there is a known
 issue between the KVM/KVM2 driver and TensorFlow Serving. The issue is tracked in [kubernetes/minikube#2377](https://github.com/kubernetes/minikube/issues/2377).
 
-Minikube by default allocates 2048Mb of RAM for its VM, however that may not align with the starting parameters for the JupyterHub server noted above. If you encounter a jupyter-xxxx pod in Pending status, described with:
+We recommend increasing the amount of resources Minikube allocates
+
 ```
-Warning  FailedScheduling  8s (x22 over 5m)  default-scheduler  0/1 nodes are available: 1 Insufficient memory.
+minikube start --cpus 4 --memory 8096 --disk-size=40g
 ```
-then try recreating your Minikube cluster (and re-apply Kubeflow using Ksonnet) with more resources (as your environment allows):
+
+  * Minikube by default allocates 2048Mb of RAM for its VM which is not enough
+    for JupyterHub.
+  * The larger disk is needed to accomodate Kubeflow's Jupyter images which
```

Comment:
Let's reword this to make clear that the size of the images are due to the ML libraries and third party packages being installed along with Jupyter. Thanks.

Reply:
Done